toolchain/check/declaration_name_stack.cpp

```diff
@@ -95,22 +95,23 @@ auto DeclarationNameStack::ApplyNameQualifierTo(NameContext& name_context,
                                                 StringId name_id) -> void {
   if (CanResolveQualifier(name_context, parse_node)) {
     // For identifier nodes, we need to perform a lookup on the identifier.
-    // This means the input node_id is actually a string ID.
+    // This means the input instruction name_id is actually a string ID.
     //
     // TODO: This doesn't perform the right kind of lookup. We will find names
     // from enclosing lexical scopes here, in the case where `target_scope_id`
     // is invalid.
-    auto resolved_node_id = context_->LookupName(
+    SemIR::InstId resolved_inst_id = context_->LookupName(
```

Comment:
Preserve `auto` for now:
```suggestion
    auto resolved_inst_id = context_->LookupName(
```